packages/react-devtools-shared/src/__tests__/profilingCache-test.js

```diff
@@ -963,7 +963,7 @@ describe('ProfilingCache', () => {
             2 => 0,
           },
           "passiveEffectDuration": null,
-          "priorityLevel": "Immediate",
+          "priorityLevel": "Normal",
```

Comment:
Why did the value change here?

Reply:
The Suspense retry runs in a microtask now instead of an Immediate Scheduler task. This is an implementation detail so it doesn't matter.